front/src/modules/scenario/components/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -105,12 +134,7 @@ const ScenarioExplorer = ({
       role="button"
       tabIndex={0}
     >
-      {globalProjectId &&
-      projectDetails &&
-      globalStudyId &&
-      studyDetails &&
-      globalScenarioId &&
-      scenarioDetails ? (
+      {globalProjectId && projectDetails && studyDetails && (scenarioV1 || scenarioV2) ? (
```

Comment:
You can directly check here if scenario is defined and remove the `?` in the next lines :)
```suggestion
      {globalProjectId && projectDetails && studyDetails && scenario) ? (
```